OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -406,14 +438,14 @@ public class ProductionItem
 		public readonly Action OnComplete;
 
 		public int TotalTime { get; private set; }
-		public int RemainingTime { get; private set; }
+		public float RemainingTime { get; private set; }
```

Comment:
Floating point numbers are not allowed in the game code.  Calculations are not deterministic, and cause hard to reproduce desyncs.

Reply:
Okay, ill work around that, any suggestion for percentual tick progress?

Reply:
Use actual percentage values, mayhaps (asin an integer fraction of 100)?